angular-rails-app/lib/tasks/db_connector.rake

```diff
@@ -150,58 +150,19 @@ namespace :db_connector do
 
   desc "Pull in hpd complaints"
   task hpd_complaints: :environment do
-
+    # TODO: missing columns: complaint_type, major_category_id, minor_category_id, code_id
```

Comment:
Ok, I think I finally tracked this down.

These columns aren't a part of `hpd_complaints`, but rather part of `hpd_complaint_problems`. Since there can be more than one problem to a complaint, we need to do some extra work to show this the right way;

* https://github.com/chrishenry/deltanyc-heatseek/issues/165
* https://github.com/chrishenry/deltanyc-heatseek/issues/166
* The design of this section needs to reflect this...somehow.